web/src/compose_validate.js

```diff
@@ -18,13 +18,18 @@ import * as people from "./people";
 import * as settings_config from "./settings_config";
 import * as settings_data from "./settings_data";
 import * as stream_data from "./stream_data";
+import * as ui from "./ui";
 import * as util from "./util";
 
 let user_acknowledged_wildcard = false;
 let wildcard_mention;
 
 export let wildcard_mention_large_stream_threshold = 15;
 
+function append_compose_banner_to_banner_list(new_row) {
+    ui.get_content_element($("#compose-content [data-simplebar]")).append(new_row)
```

Comment:
Overbroad selector. Use a specific ID or class name *for the element you are selecting*, not one of its ancestors. Don’t be afraid to add one to the HTML if you need to. In this case you don’t: it’s just `#compose_banners`.

Reply:
Agreed! I'm not sure why I used such a broad selector. `#compose_banners` should suffice. Thank you for your help!